Sources/TuistDependencies/Carthage/Utils/CarthageGraphGenerator.swift

```diff
@@ -28,17 +28,30 @@ public final class CarthageGraphGenerator: CarthageGraphGenerating {
 
         let thirdPartyDependencies: [String: ThirdPartyDependency] = Dictionary(grouping: products, by: \.name)
             .compactMapValues { product in
-                guard let frameworkName = product.first?.container else { return nil }
-
-                let path = AbsolutePath("/")
-                    .appending(components: [
-                        Constants.tuistDirectoryName,
-                        Constants.DependenciesDirectory.name,
-                        Constants.DependenciesDirectory.carthageDirectoryName,
-                        frameworkName,
-                    ])
-
-                return .xcframework(path: path)
+                if let xcFrameworkName = product.first?.container {
```

Comment:
I would do a `guard let product = product.first else { return nil }` instead of doing optional binding in both branches, so the second branch can become a simple `else`

Reply:
That's fixed, thanks!